### PR TITLE
unfork pcplod

### PR DESCRIPTION
**Pull request by SethTisue**
as per @fommil's request at https://github.com/scala/community-builds/issues/475#issuecomment-304481026

### CODE REVIEW

**Comment by SethTisue** [comment]
test run queued: https://scala-ci.typesafe.com/job/scala-2.12.x-integrate-community-build/1537/consoleFull (404 til Jenkins finishes waxing his slide rule)

File: configs/project-refs.conf [hunk 1]
```diff
@@ -49,7 +49,7 @@ vars.uris: {
   nyaya-uri:                    "https://github.com/japgolly/nyaya.git"
   parboiled-uri:                "https://github.com/sirthias/parboiled.git#community-build-2.12"
   parboiled2-uri:               "https://github.com/sirthias/parboiled2.git#release-2.1"
-  pcplod-uri:                   "https://github.com/scalacommunitybuild/pcplod.git#community-build-2.12"  # was master
+  pcplod-uri:                   "https://github.com/scalacommunitybuild/pcplod.git"
```
Comment (fommil) [inline configs/project-refs.conf]: @SethTisue you forgot to change the URI back to my repo (or to sync your repo)

Comment (SethTisue) [reply configs/project-refs.conf]: doh! thx

**Comment by SethTisue** [comment]
new test run: https://scala-ci.typesafe.com/job/scala-2.12.x-integrate-community-build/1538/consoleFull

**Comment by SethTisue** [comment]
@fommil the log shows this failure:

```
[spray-json-shapeless] [error] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.7-RC1/project-builds/spray-json-shapeless-b606823af7da707f9c5c036d75af48805d3e52c2/src/test/scala/fommil/sjs/FamilyFormatsSpec.scala:354: could not find implicit value for evidence parameter of type spray.json.JsonFormat[fommil.sjs.ExampleAst.TokenTree]
[spray-json-shapeless] [error]     roundtrip(SpecialToken: TokenTree, """{"type":"SpecialToken"}""")
[spray-json-shapeless] [error]              ^
[spray-json-shapeless] [error] one error found
[spray-json-shapeless] [error] (test:compileIncremental) Compilation failed
```

same error as we started with over at https://github.com/scala/community-builds/issues/475

closing, but we can reopen if or when you think you might have a fix

**Comment by SethTisue** [comment]
I'll make a separate PR that just unforks pcplod.

**Comment by SethTisue** [comment]
on second thought: I'll get rid of the spray-json-shapeless part of this and merge the pcplod part.

**Comment by fommil** [comment]
I'm pretty sure this is a regression in shapeless because it passes fine with 2.12.2 and it has been happening since before 2.12.2 was cut. @milessabin have there been any changes in shapeless `master` that could impact typeclass derivation of sealed families?

**Comment by fommil** [comment]
specifically, typeclass derivation of this AST https://github.com/fommil/spray-json-shapeless/blob/master/src/test/scala/fommil/sjs/ExampleAst.scala

**Comment by SethTisue** [comment]
I'd suggest continuing on https://github.com/scala/community-builds/issues/475, where the previous discussion is

**Pull request merged**